<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>debugDockWidget</class>
 <widget class="QDockWidget" name="debugDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Debug View</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>501</width>
         <height>307</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Set position</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QDoubleSpinBox" name="xPosDoubleSpinBox">
             <property name="minimum">
              <double>-9999.000000000000000</double>
             </property>
             <property name="maximum">
              <double>9999.989999999999782</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="yPosDoubleSpinBox">
             <property name="minimum">
              <double>-9999.000000000000000</double>
             </property>
             <property name="maximum">
              <double>9999.989999999999782</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="zPosDoubleSpinBox">
             <property name="minimum">
              <double>-9999.000000000000000</double>
             </property>
             <property name="maximum">
              <double>9999.989999999999782</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="radiusDoubleSpinBox">
          <property name="minimum">
           <double>-99.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rebuildAABBPushButton">
          <property name="text">
           <string>Rebuild AABB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Find end points test</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>2</number>
           </property>
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Selection</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QSpinBox" name="cSelSpinBox">
                <property name="maximum">
                 <number>700</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="copySelectedToCPushButton">
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>C</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="aSelSpinBox">
                <property name="maximum">
                 <number>700</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="copySelectedToAPushButton">
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>C</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="bSelSpinBox">
                <property name="maximum">
                 <number>700</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="copySelectedToBPushButton">
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>C</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="testIntersectionPushButton">
             <property name="text">
              <string>Find end points</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="extractSurfaceDebugPushButton">
          <property name="text">
           <string>Extract surface selected</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
